### PR TITLE
Built mobile header and headline

### PR DESCRIPTION
**Pull request by ONJoseph**
Project setup

### CODE REVIEW

**Review by nicupop729 — COMMENTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @ONJoseph , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that fails the linter checks.
And please add `.gitignore` file with `node_modules/` folder in it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# Portfolio
```
Comment (ONJoseph) [inline README.md:1]: edit

File: index.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="headline-ilustration">
+            <img src="images/Illustration-header.svg" class="headline-img" alt="ilustration image">
+        </div>
+    </section>
+    <section class="works">
+        <div class="works-title">
+            <h3 class="work-title">My Recent Works</h3>
```
Comment (codepantherr) [inline index.html:60]: - [ ] Even though you have done an amazing job, this section down to the footer is outside the scope of this project for now. Kindly remove ❌  them as they are not required at this stage of the project.

**Review by codepantherr — CHANGES_REQUESTED** [review]
Hi @ONJoseph 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## To Highlight: ⚡ 
- Descriptive Readme
- Meaningful PR

## Suggested changes
- [ ] Please remove all unused CSS code. Most of the code you have there are not required at this stage of the project.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**